client/my-sites/upgrades/verify-email-forward/verify-email-forward.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External Dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+
+class VerifyEmailForward extends React.Component {
+	render() {
+		const { translate, mailbox, destination } = this.props,
+			title = translate( 'Verify Email Forwarding', { context: 'Header' } );
+		return (
+			<Main wideLayout>
+				<DocumentHead title={ title } />
+				<SectionHeader label={ title } />
+				<Card>
+					<p>
+						{ translate(
+							'We will forward all emails arriving to {{strong}}%(inboxEmail)s{{/strong}} ' +
+							'to {{strong}}%(destinationEmail)s{{/strong}}.', {
+								args: {
+									inboxEmail: mailbox,
+									destinationEmail: destination
+								},
+								components: {
+									strong: <strong />
+								}
+							} ) }
+					</p>
+					<FormButton>
+						{ title( 'Continue' ) }
```

Comment:
This (and line 44) cause an error:
```
TypeError: title is not a function
   at VerifyEmailForward.render (/Users/klimeryk/work/workspace/calypso/build/webpack:/client/my-sites/upgrades/verify-email-forward/verify-email-forward.jsx:41:9)
...
```
You were probably thinking of `translate` (damn compiler not being smart enough! we should write a babel plugin to fix this! ;))